### ISSUE TITLE
Linux Ubuntu 21.04 PS3 controller left right analog not work

### ISSUE THREAD

**Issue by Augusto7743** [stella-emu/stella#797]
Linux Ubuntu 21.04 64 bits and PS3 controller Stella 6.5.3 64 bits.
The controller works with anothers emulators.
In Stella when configuring input for player 1 is possible map any input using digital pad and buttons , but not is possible map left and right analog.
When click in map for player 1 control and moving left analog is how if Stella not is reading controller input.
In anothers emulators works.
Is an issue or need to configure input driver (evdev or other) in stella cfg ?

**Comment by thrust26** [comment]
Have you tried moving to the very left and right while mapping?

**Comment by Augusto7743** [comment]
tried totally to both sides not work
others emulators has access to both leftand right analog sticks.
look how if stella not is reading data from controller.
hidapi is installed and other emulators use evdev.

**Comment by thrust26** [comment]
And did you check #767 and with a reduced deadzone?

**Comment by Augusto7743** [comment]
Yes ... I had used the software evdev-joystick.tar.xz from https://stella-emu.github.io/downloads.html and configured to use deadzone lowest value.
That not is the PS3 sixaxis values creating mess ? if moving the controller to press an button also is created an sixaxis values of controller position.
The controller works with anothers emulators , but i see that in few times I need to shake the controller to L3 and R3 work again because is how if temporary not read both buttons because sixaxis.
Here Stella is the only that not read the analog stick left and right.

**Comment by thrust26** [comment]
@sa666666 Do you happen to have such a controller? I am running out of ideas.
